tests/py/unit/test_build_atomic_layers.py

```diff
@@ -0,0 +1,53 @@
+from mat3ra.made.tools.analyze.lattice_planes import CrystalLatticePlanesMaterialAnalyzer
+from mat3ra.made.tools.analyze.other import get_local_extremum_atom_index
+from mat3ra.made.tools.build.slab.builders import AtomicLayersUniqueRepeatedBuilder, CrystalLatticePlanesBuilder
+from mat3ra.made.tools.build.slab.configuration import (
+    AtomicLayersUniqueRepeatedConfiguration,
+    CrystalLatticePlanesConfiguration,
+)
+from mat3ra.made.tools.build.slab.helpers import select_slab_termination
+from unit.fixtures.generated.fixtures import SrTiO3_BULK_MATERIAL
+
+NUMBER_OF_LAYERS = 1
+
+
+def process_termination(material, miller_indices, termination):
+    crystal_lattice_planes_config = CrystalLatticePlanesConfiguration(crystal=material, miller_indices=miller_indices)
+    crystal_lattice_planes_material = CrystalLatticePlanesBuilder().get_material(crystal_lattice_planes_config)
+
+    atomic_layers_config = AtomicLayersUniqueRepeatedConfiguration(
+        crystal=crystal_lattice_planes_material,
```

Comment:
Why are we passing crystal_lattice_planes_material here? Should be just `material`